docs/tutorial/image_det.md

```diff
@@ -16,18 +16,18 @@ Blueoil supports 2 formats for object detection.
 Note: *Please see the detail in <a href="../usage/dataset.html">Prepare training dataset</a>*
 
 You can download subset of Open Images V4 from
-[our server](https://s3-ap-northeast-1.amazonaws.com/leapmind-public-storage/datasets/openimages.tgz).
+[our server](https://s3-ap-northeast-1.amazonaws.com/leapmind-public-storage/datasets/openimages_face.tgz).
 
 
-	$ wget https://s3-ap-northeast-1.amazonaws.com/leapmind-public-storage/datasets/openimages.tgz
-	$ tar xf openimages.tgz
+	$ wget https://s3-ap-northeast-1.amazonaws.com/leapmind-public-storage/datasets/openimages_face.tgz
+	$ tar xf openimages_face.tgz
 
 
-This dataset consists of 5,000 color images in 2 classes, with 2,500 images per class.
+This dataset consists of 2869 Human Face images and 5171 annotation boxes.
```

Comment:
I think number of images and boxes is wrong.

I download openimages_face.tgz and extract it.

```
$ find ./images/ -type f -name "*.jpg" | wc -l
    2866
```
The number of images is `2866`.


```
$ cat annotations-bbox.csv | wc -l
    5171
```
`annotations-bbox.csv` is including header, so the number boxes is `5170 (5171 - 1)`

Reply:
Thanks. I fixed.